### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by sabrasisler**
Notifications page is currently working. Everything should be documented, and the tests are in the works. This PR covers issue #49, issue #52, #22. It renders all three types of notifications differently, changes the appearance of a notification when it is viewed (aka clicked on), and shows only the notifications for a given user.

### CODE REVIEW

**Review by spencersolomon6 — APPROVED** [review]
Looks great!

**Review by tmudunuri — APPROVED** [review]

File: src/components/Notifications/index.js
Comment (creativediego) [inline src/components/Notifications/index.js]: check for notifications existence is being done twice. The ternary is redundant.

File: src/components/Notifications/notification.js
Comment (creativediego) [inline src/components/Notifications/notification.js]: const or let is more standard Javascript for variable declaration; var has become anti-pattern. Could we rename nstring to something more descriptive?

File: src/components/Notifications/notification.js
Comment (creativediego) [inline src/components/Notifications/notification.js]: This if/else might grow quite a bit hypothetically if we add more types of notifications or turn into a long switch. How about an object literal/dictionary object with the type as a key and value as markup?

File: src/views/NotificationsView/NotificationsView.js
Comment (creativediego) [inline src/views/NotificationsView/NotificationsView.js:20]: Non-standard use of useState. Also, to be more consistent, where we return setError, we could dispatch a redux action to update the error state in the entire application, as such:
Instead of ===> const [setError] = useState()
We do ===> const dispatch = useDispatch()
Then in the return statement ====> dispatch(updateGlobalError(errorStringHere);
We'd import the methods at the top. See the file "MockInbox" for an example.

**Review by creativediego — CHANGES_REQUESTED** [review]
Very nice implementation of the service, and I like that you stay consistent with the overall style. Just needs a few minor changes.

File: src/views/NotificationsView/NotificationsView.js
Comment (sabrasisler) [reply src/views/NotificationsView/NotificationsView.js:20]: I can try to change it, but looking at the mockInbox didn't really help, I would need you to elaborate more. We already have PRs that contain the same logic that I used, so I would prefer put this as a lower style priority until next week so I can merge this PR. Until this is PR is merged, I am blocked on some of my other tasks.

File: src/components/Notifications/notification.js
Comment (sabrasisler) [reply src/components/Notifications/notification.js]: Renamed to notifications string. I wanted to use notifications message, but thought that might be confusing regarding the distinction between notifications message type.

**Pull request merged**
